go/ct/spc/specification.go

```diff
@@ -1334,6 +1334,11 @@ func getAllRules() []Rule {
 
 	// --- EXTCODEHASH ---
 
+	extCodeHashEffect := func(s *st.State, address tosca.Address) {
```

Comment:
nit: popping the address from the stack in the state could be also moved into this function.